pims/bioformats.py

```diff
@@ -5,32 +5,53 @@
 from pims.base_frames import FramesSequence
 from pims.frame import Frame
 from warnings import warn
-from os.path import isfile
+import os
 
 try:
-    import javabridge
+    import jpype
 except ImportError:
-    javabridge = None
+    jpype = None
 
-try:
-    import bioformats
-except ImportError:
-    bioformats = None
+LOCI_TOOLS_PATH = os.path.join(os.path.dirname(__file__),
```

Comment:
Ok I'll work on some search cascade. It's not that I don't like it because it's complicated, I just don't like it because complicates often means more prone to errors.
But I'll start form the metadatastore conf file you linked.